README.md

```diff
@@ -197,23 +197,27 @@ Any value in the configuration language can be a variable reference (`key=$VALUE
 
 ## Clustering
 
-NATS supports running each server in clustered mode. You can cluster servers together for high volume messaging systems and resiliency and high availability. Clients are cluster-aware. See also the [clustered NATS](http://nats.io/documentation/server/gnatsd-cluster/) documentation.
+Clustering lets you scale NATS messaging by having multiple NATS servers communicate with each other. Clustering lets messages published to one server be routed and received by a subscriber on another server. See also the [clustered NATS](http://nats.io/documentation/server/gnatsd-cluster/) documentation.
 
-The following cluster options are supported:
+### Full mesh recommended
```

Comment:
yes please

On Thu, Jul 14, 2016 at 3:01 PM, Alberto Ricart <notifications@github.com>
wrote:

> In README.md
> <https://github.com/nats-io/gnatsd/pull/308#discussion_r70892372>:
>
> >
> > -The following cluster options are supported:
> > +### Full mesh recommended
>
> Lorenzo do you want me to correct this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nats-io/gnatsd/pull/308/files/dbeaad02a6917dd0c852e21f0a0166b6f5dfb30b#r70892372>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJV02UUqcRvKd1lgD9B10iQ19QEkGQQmks5qVrGtgaJpZM4JDcqN>
> .
>



-- 
*Lorenzo Paris* | Senior Manager, Documentation & Training
lparis@apcera.com | 415.609.4850

[image: Why Apcera?] <https://www.apcera.com/why-apcera>

<https://www.apcera.com/why-apcera>